lib/config/rfc-status-hierarchy.mjs

```diff
@@ -0,0 +1,58 @@
+export const rfcStatusHierarchy = [
+  {
+    name: 'Internet Standard',
+    regex: /internet standard/ig,
+    weight: 7
+  },
+  {
+    name: 'Draft Standard',
+    regex: /draft standard|ds/ig,
+    weight: 6
+  },
+  {
+    name: 'Proposed Standard',
+    regex: /proposed standard|ps/ig,
+    weight: 5
+  },
+  {
+    name: 'Standards Track',
+    regex: /standards track|std/ig,
+    weight: 5
+  },
+  {
+    name: 'Best Current Practice',
+    regex: /best current practice|bcp/ig,
+    weight: 4
```

Comment:
I'm still concerned about this weight concept. Where did these come from? I think they're not helpful.

A normative reference from a standards track document to a BCP is not a downref and vice-versa.
A normative reference from an Informational document to an Experimental document isn't a downref or vice-versa.

Downrefs only apply to normative references made from standards track documents (proposed standard, draft standard, and internet standard) or from BCPs. (so a normative reference from a BCP to an Experimental or Informational document would be a downref.)

This tool should also call out any normative reference from a standards track or BCP document to a document that is currently Historic (even if it that historic document was once standards track) as a downref.